resources/Microsoft.Windows.Setting.Accessibility/Microsoft.Windows.Setting.Accessibility.psm1

```diff
@@ -288,9 +304,20 @@ class VisualEffect
             {
                 New-Item -Path $global:ControlPanelAccessibilityRegistryPath -Force | Out-Null
             }
+            if (-not (Test-Path -Path $global:ControlPanelAccessibilityRegistryPath))
+            {
+                New-Item -Path $global:AudioRegistryPath -Force | Out-Null
+            }
+
+            if (-not (DoesRegistryKeyPropertyExist -Path $global:AudioRegistryPath -Name $this.EnableMonoAudioProperty)) {
+                New-ItemProperty -Path $global:AudioRegistryPath -Name $this.EnableMonoAudioProperty -Value $desiredState -PropertyType DWord
+            }
 
             $dynamicScrollbarValue = $this.AlwaysShowScrollbars ? 0 : 1
+            $monoAudioValue = $this.EnableMonoAudio ? 0 : 1

```

Comment:
Worth noting that ternaries are only valid in PowerShell, not Windows PowerShell. If you want this resource to be usable in Windows PowerShell (or, more importantly, if your module doesn't _require_ v7+), this will need to be replaced.

Reply:
cc @ryfu-msft

Reply:
Removed ternary operator and used a normal if condition.